core/services/bulletprooftxmanager/eth_confirmer.go

```diff
@@ -171,11 +174,22 @@ func (ec *EthConfirmer) processHead(ctx context.Context, head models.Head) error
 	logger.Debugw("EthConfirmer: finished RebroadcastWhereNecessary", "headNum", head.Number, "time", time.Since(mark), "id", "eth_confirmer")
 	mark = time.Now()
 
-	defer func() {
-		logger.Debugw("EthConfirmer: finished EnsureConfirmedTransactionsInLongestChain", "headNum", head.Number, "time", time.Since(mark), "id", "eth_confirmer")
-	}()
+	if err := errors.Wrap(ec.EnsureConfirmedTransactionsInLongestChain(ctx, head), "EnsureConfirmedTransactionsInLongestChain failed"); err != nil {
+		return err
```

Comment:
a bit minor, but it would be more readable and consistent with other places to have `errors.Wrap` on the `return` line - what do you think?